Source/WebCore/platform/graphics/cg/ImageUtilitiesCG.cpp

```diff
@@ -164,4 +166,95 @@ Expected<std::pair<String, Vector<IntSize>>, ImageDecodingError> utiAndAvailable
     return std::make_pair(WTFMove(uti), WTFMove(sizes));
 }
 
+Vector<uint8_t> createIconFromImageData(std::span<const uint8_t> data, std::span<const unsigned> lengths)
```

Comment:
Can we be explicit about this function is returning? Since we are returning `IconData` not `Icon` and since we do not know if the passed data is an ImageData or just `Data`, can't we use something like `createIconDataFromData()`?